### PR TITLE
Adjust min-height of the program list container

### PR DESCRIPTION
**Pull request by d8660091**
**Description of changes**

Use container's loading spinner to keep the spinner centered

**Type of Change**

- [ ] Bug
- [x] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/submission-system/programs/index.tsx [hunk 1]
```diff
@@ -102,14 +102,17 @@ export default function Programs({ authorizedPrograms = [] }: any) {
         id="programs-list-container"
         css={css`
           padding-top: 0px;
+          min-height: 300px;
```
Comment (hlminh2000) [inline components/pages/submission-system/programs/index.tsx]: do we maybe want this in every container and override on a per-case basis? @kcullion

Comment (kcullion) [reply components/pages/submission-system/programs/index.tsx]: @hlminh2000 you mean set the min-height to 300px then overwrite for certain containers? 
That's what we thought..I think @d8660091 suggested something different. What do we think is easiest going forward?

Comment (hlminh2000) [reply components/pages/submission-system/programs/index.tsx]: yeah that was my understanding too, and I'm ok with that. I think there's probably more use-case with the min-height than without, so would rather do it at the base component.

Comment (hlminh2000) [reply components/pages/submission-system/programs/index.tsx]: ok a quick discussion with @kcullion and scanning through the ticket, we decided that it's good to apply this at the per-case basis. Sorry I was thrown off by the ticket title :P

**Pull request merged**